specification/dartLangSpec.tex

```diff
@@ -15862,7 +15866,11 @@ \subsection{Lexical Lookup}
 
 \LMHash{}%
 A lexical lookup yields a result which is
-a declaration, an import prefix, or a member signature.
+a declaration, an import prefix,
+or the special value
+\Index{\PrependThisValue},
+which indicates that the lookup should be transformed into
+a member access on \THIS{}.
 A lexical lookup can not fail
```

Comment:
This is a really confusing comment.  It most certainly can fail, right?  What does it mean to say that a compile-time error can occur but it cannot fail?

Reply:
I rewrote this to emphasize the handling of lexical lookups, avoiding the words "cannot fail".